tests/utils/mock-execa.js

```diff
@@ -0,0 +1,30 @@
+const tempy = require('tempy')
+
+const { rmdirRecursiveAsync, writeFileAsync } = require('../../src/lib/fs')
+
+// Saves to disk a JavaScript file with the contents provided and returns
+// an environment variable that replaces the `execa` module implementation.
+// A cleanup method is also returned, allowing the consumer to remove the
+// mock file.
+const createMock = async (contents) => {
+  const path = tempy.file({ extension: 'js' })
+
+  await writeFileAsync(path, contents)
+
+  const env = {
+    NETLIFY_CLI_EXECA_PATH: path,
+  }
+  const cleanup = () => removeMock(path)
+
+  return [env, cleanup]
+}
+
+const removeMock = async (path) => {
+  try {
+    await rmdirRecursiveAsync(path)
```

Comment:
[sand] Should this pattern (noop `try`/`catch` + `rmdirRecursiveAsync()`) be moved to a new utility function in `./src/lib/fs`?

Reply:
Done in https://github.com/netlify/cli/pull/3047/commits/38a6c6a11e29462e2ac1d227339c295c7fcd5cb9.